### PR TITLE
docs(material/card): update docs & examples for MDC-based card

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

**Comment by mmalerba** [comment]
@jelbourn the last commit contains just the markdown changes if you wanted to look at those

File: src/material/card/testing/card-harness.e2e.spec.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {browser} from 'protractor';
 describe('card harness', () => {
   let loader: HarnessLoader;
 
-  beforeEach(async () => await browser.get('/mdc-card'));
+  beforeEach(async () => await browser.get('/cards'));
```
Comment (crisbeto) [inline src/material/card/testing/card-harness.e2e.spec.ts:9]: nit: shouldn't this be `card`?

**Review by crisbeto — APPROVED** [review]
LGTM

File: src/material/card/card.md [hunk 2]
```diff
@@ -2,38 +2,55 @@
...
+| `<mat-card-footer>`      | Section anchored to the bottom of the card                     |
 
 These elements primary serve as pre-styled content containers without any additional APIs. 
 However, the `align` property on `<mat-card-actions>` can be used to position the actions at the 
-`'start'` or `'end'` of the container.  
+`'start'` or `'end'` of the container.
+
+### Card padding
 
+The `<mat-card>` element itself does not add ay padding around its content. This allows developers
```
Comment (Ryczko) [inline src/material/card/card.md:24]: typo: `ay padding`

File: src/material/card/testing/card-harness.e2e.spec.ts (hunk 1 above)
Comment (mmalerba) [reply src/material/card/testing/card-harness.e2e.spec.ts:9]: For some reason no, its /cards in the e2e app

File: src/material/card/card.md (hunk 2 above)
Comment (mmalerba) [reply src/material/card/card.md]: good catch, fixed

**Pull request merged**